### ISSUE TITLE
`npm config` edit throwns

### ISSUE THREAD

**Issue by liquid** [npm/npm#5581]
I've been trying to edit the NPM config as per the `npm help config` docs, but looks like the config file cannot be found:

```
➜ ~ (home) ✗ npm config get loglevel
warn
➜ ~ (home) ✗ npm config edit --global
npm ERR! Error: spawn ENOENT
npm ERR!     at errnoException (child_process.js:1000:11)
npm ERR!     at Process.ChildProcess._handle.onexit (child_process.js:791:34)
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Darwin 13.2.0
npm ERR! command "node" "/usr/local/bin/npm" "config" "edit" "--global"
npm ERR! cwd /Users/attila
npm ERR! node -v v0.10.29
npm ERR! npm -v 1.4.16
npm ERR! syscall spawn
npm ERR! code ENOENT
npm ERR! errno ENOENT
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/attila/npm-debug.log
npm ERR! not ok code 0
```

Where am I supposed to look for the config file? I don't have a `$HOME/.npmrc` file by the way, yet npm correctly remembered the configuration setting.

**Comment by smikes** [comment]
Is this still a problem for you?

It looks like you were getting an ENOENT on the `spawn` call, which might mean that the editor specified by your $EDITOR or $VISUAL environment variable did not exist.

I note that if I run `npm config edit --global` and the path to the expected `npmrc` file does not exist, then I get an error, but this is a separate issue.

```
$ npm config edit --global
npm ERR! Darwin 14.1.0
npm ERR! argv "node" "/Users/smikes/.nvm/versions/io.js/v1.0.4/bin/npm" "config" "edit" "--global"
npm ERR! node v1.0.4
npm ERR! npm  v2.3.0
npm ERR! path /Users/smikes/.nvm/versions/io.js/v1.0.4/etc/npmrc.c33db61dc6e13a89a2ab11f8f1793b9e
npm ERR! code ENOENT
npm ERR! errno -2

npm ERR! enoent ENOENT, open '/Users/smikes/.nvm/versions/io.js/v1.0.4/etc/npmrc.c33db61dc6e13a89a2ab11f8f1793b9e'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/smikes/src/github/npm-kludge-search/npm-debug.log
```

We are trying to clean up older npm issues, so if we don't hear back from you within a week, we will close this issue. (Don't worry -- you can always come back again and open a new issue!)

Thanks!

**Comment by othiym23** [comment]
Closing as resolved / abandoned. I believe this as at least partially addressed by 6fd0fbd8a0347fd47cb7ee0064e0902a2f8a087c, so upgrade to `npm@2.5.0` or newer if you run across this issue in the future and see if that fixes the problem.
